circle.yml

```diff
@@ -206,6 +168,7 @@ jobs:
       - *save-eth-cache
       - *test-evm2wasm
 
+  # to run this using local circleci tool, rename Ewasm-Tests to `build` then do `circleci build -c circle.yml`
```

Comment:
Is this accurate when `ewasm-tests` is removed?

Reply:
I don't have the `circleci` commandline tool on the mac, not sure how it works and why cannot it point to a workflow task (instead of build)?

Reply:
Remove the comment. Seems obsolete.